service-loadbalancer/Dockerfile

```diff
@@ -16,10 +16,12 @@
 FROM alpine:3.2
 MAINTAINER Prashanth B <beeps@google.com>
 
-RUN apk add -U haproxy bash curl && \
+RUN apk add -U pcre bash curl socat && \
+  wget -O /var/cache/apk/haproxy.apk https://github.com/aledbf/alpine-build-dev-haproxy/releases/download/v1.6-dev6/haproxy-1.6-rdev6.apk && \
```

Comment:
can we download this from http://www.haproxy.org/? 
As a bonus if you're going to compile it from official, please enalbe the lua engine (http://blog.haproxy.com/2015/10/14/whats-new-in-haproxy-1-6/), I'm sure it'll come in handy ;)